src/lib/crypto/libk5crypto.exports

```diff
@@ -83,9 +83,9 @@ krb5_k_make_checksum_iov
 krb5_k_prf
 krb5_k_reference_key
 krb5_k_verify_checksum
-krb5_k_verify_checksum_iov
 krb5int_aes_encrypt
 krb5int_aes_decrypt
+krb5_k_verify_checksum_iov
```

Comment:
This movement seems unrelated?

Reply:
I tried removing krb5int_aes_{enc,dec}rypt (which are used by a test program, it turns out) and accidentally put them back at a one-line offset.  Fixed along with the commit message.